### PR TITLE
refactor(COSMOTE TV): recode & improve functionality

### PR DESCRIPTION
**Pull request by down-bad**
## Description 

Rewrote the presence to improve the code. Also: improved series info and timestamps, channel logos now apply from the image src, and added settings.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</img>
</img>
</img>

</details>

### CODE REVIEW

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/C/Cosmote TV/presence.ts [hunk 1]
```diff
@@ -5,108 +5,193 @@ const presence = new Presence({
...
+					)
+					?.src.replace("-wide", "-normal");
+			}
 
-		if (!live) {
-			presenceData.smallImageKey = paused ? "pause" : "play";
-			presenceData.smallImageText = paused
-				? (await strings).pause
-				: (await strings).play;
+			// Live
```
[the remark this answers is not in the document]
Comment (down-bad) [reply websites/C/Cosmote TV/presence.ts:104]: Thanks for your suggestions

**Review by EncryptedDev — APPROVED** [review]
Good PR

File: websites/C/Cosmote TV/presence.ts [hunk 2]
```diff
@@ -5,108 +5,193 @@ const presence = new Presence({
...
+						),
+						presence.timestampFromFormat(
+							document.querySelector<HTMLSpanElement>("#Vduration").textContent
+						)
+					);
+
+				presenceData.smallImageKey = paused ? "pause" : "play";
+				presenceData.smallImageText = paused
+					? (await strings).pause
+					: (await strings).play;
```
Comment (Bas950) [inline websites/C/Cosmote TV/presence.ts:131]: await the strings at the top of the file instead. @EncryptedDev for more info.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/C/Cosmote TV/presence.ts (hunk 2 above)
Comment (EncryptedDev) [reply websites/C/Cosmote TV/presence.ts:131]: Like thishttps://github.com/PreMiD/Presences/blob/b622198bdf2396867a11ed644a3725d06e2a6745/websites/S/Skribbl.io/presence.ts#L14

**Review by Bas950 — APPROVED** [review]

**Pull request merged**